src/parseSegments.spec.ts

```diff
@@ -256,6 +256,32 @@ test('failure when splat is not at the end', () => {
   );
 });
 
+test('failure when escape sequence contains a slash', () => {
+  expect(() => parseSegments('u[/]ers')).toThrowErrorMatchingInlineSnapshot(
+    `
+    [Error: Failed to parse segments
+      'u[/]ers'
+         ^ expected anything except "/"]
+  `,
+  );
+
+  expect(() => parseSegments('u[/s]ers')).toThrowErrorMatchingInlineSnapshot(
+    `
+    [Error: Failed to parse segments
+      'u[/s]ers'
+         ^ expected anything except "/"]
+  `,
+  );
+
+  expect(() => parseSegments('u[s/]ers')).toThrowErrorMatchingInlineSnapshot(
+    `
+    [Error: Failed to parse segments
+      'u[s/]ers'
+          ^ expected ']']
+  `,
+  );
```

Comment:
These errors seem inconsistent. In my opinion, the last one should also say `expected anything except "/"`